erpnext/stock/doctype/quality_inspection/quality_inspection.py

```diff
@@ -145,3 +146,36 @@ def postprocess(source, doc):
 	}, target_doc, postprocess)
 
 	return doc
+
+@frappe.whitelist()
+def get_purchase_item_details(doctype, name, item_code):
+	doc = frappe.get_doc(doctype, name)
+	for item in doc.items:
+		if item.item_code == item_code:
+			data = {
+				"supplier": doc.supplier,
+				"uom": item.uom,
+				"qty": item.qty
+			}
+			return data
```

Comment:
Where did this come from?

Reply:
after rebase

Reply:
I think @bhattdevarsh  worked on it

Reply:
Then it shouldn't be added in your changes, this probably already exists then